kpi-tracker/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "concordium-kpi-tracker"
-version = "2.1.3"
+version = "2.1.4"
```

Comment:
Please update changelog or leave the old version (since we didn't push out a docker images yet with the old version).

Reply:
The tag cannot be deleted because of a ruleset. Rather just bump it